src/bidiServer/index.ts

```diff
@@ -75,7 +75,12 @@ function parseArguments() {
   try {
     log('Launching BiDi server');
 
-    const args = parseArguments();
+    const args: {
```

Comment:
I think having it on the function as return type is a little bit cleaner, WDYT?

Reply:
Agreed, done.